app/templates/CONTRIBUTING.md

```diff
@@ -40,11 +40,12 @@ To be consistent, we require commit messages to be in this specific format: `<ty
   * test (when adding missing tests)
   * chore (maintain)
 * Scope: anything that specifies the scope of the commit. Can be blank or `*`
+* If your commit addresses a specific issue, include the number after the type
```

Comment:
👍 
Maybe include a link to all issues (https://github.com/screwdriver-cd/screwdriver/issues)